python/hail/dataset.py

```diff
@@ -1615,6 +1615,9 @@ def export_vcf(self, output, append_to_header=None, parallel=False):
         Sets and Arrays are output with the same comma-separated format.
         Lastly, fields of type Boolean are allowed in ``va.info`` to generate INFO fields of type Flag.
 
+        Hail also exports the name of the genome reference (e.g. ``GRCh37``), as well as contig names and lengths,
```

Comment:
I don't think we need this in the documentation. What do you think?

Reply:
Actually, maybe it's fine. But can you reword it so the assembly in contig field is set to genome reference name.

Reply:
How's this?